resources/migrations/000_migrations.yaml

```diff
@@ -15159,6 +15159,51 @@ databaseChangeLog:
               column:
                 name: action_id
 
+  - changeSet:
+      id: v48.00-007
+      author: qnkhuat
+      comment: Added 0.48.0 - Add revision.most_recent
+      changes:
+        - addColumn:
+            tableName: revision
+            columns:
+              - column:
+                  name: most_recent
+                  type: boolean
+                  defaultValueBoolean: false
+                  remarks: 'Whether a revision is the most recent one'
+                  constraints:
+                    nullable: false
```

Comment:
Creating an index for this should speed up the search query for large instances. Not sure how significant it will be though.

Reply:
yep, created an index here https://github.com/metabase/metabase/pull/33405/commits/91b0bf3d964a9c851e036d8b7bf462461d66cab1